frontend/public/components/chargeback.tsx

```diff
@@ -282,31 +272,40 @@ class ReportData extends React.Component<ReportDataProps, ReportDataState> {
         "pod_request_memory_byte_seconds": 23385341952000
       },
       All fields but start/end are optional. */
-    _.each(data, row => {
-      const key = row[reduceBy];
-      // key == 'namespace', 'pod', 'node' (whatever we're aggregating by)
-      if (!reducedData[key]) {
-        reducedData[key] = {};
-      }
-      _.each(row, (v, k) => {
-        if (!isFinite(v as (any))) {
-          return;
-        }
-        // k == 'pod_memory_usage_percent', 'cost', etc (all columns in table except 1st)
-        if (!reducedData[key][k]) {
-          reducedData[key][k] = 0;
-        }
-        reducedData[key][k] += v;
-        if (!totals[k]) {
-          totals[k] = 0;
-        }
-        totals[k] += v;
-        if (reducedData[key][k] > (maxValues[k] || 0)) {
-          maxValues[k] = reducedData[key][k];
+
+    const canReduceData:boolean = !_.isEmpty(data) && _.get(data, [0, reduceBy]) !== undefined;
```

Comment:
TypeScript can infer the boolean type.

```suggestion
    const canReduceData = !_.isEmpty(data) && _.get(data, [0, reduceBy]) !== undefined;
```

Reply:
fixed